src/components/Person/Person.jsx

```diff
@@ -1 +1,42 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Person({ person }) {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  const partnerStatus = sex === 'f' ? 'husband' : 'wife';
+  const relationshipStatus = isMarried
+    ? (<p className="Person__partner">{`${partnerName} is my ${partnerStatus}`}</p>)
```

Comment:
IMO, it's better to do all conditional rendering inside JSX, because now I need to jump between variables and remember what will be rendered and where 